plugins/blueprint.cpp

```diff
@@ -755,16 +756,12 @@ command_result blueprint(color_ostream &out, vector<string> &parameters)
     DFCoord start(options.start);
     if (options.start.x == -30000)
```

Comment:
Just noticed that `df::coord` has an [`isValid()` method](https://github.com/DFHack/dfhack/blob/41ff2c2e6c612814e09d03704f00bcdfb613db52/library/include/df/custom/coord.methods.inc#L6) that does this too. I don't know how widely it's used - I know there are a _lot_ of hardcoded checks for `-30000` too - but if you end up making other changes, this could be something to use as well.

(This PR is still on my list. Hopefully tomorrow!)